Zend/zend_enum.c

```diff
@@ -207,29 +207,61 @@ static ZEND_NAMED_FUNCTION(zend_enum_cases_func)
 	} ZEND_HASH_FOREACH_END();
 }
 
+ZEND_API zend_object *zend_enum_get_case_by_value(zend_class_entry *ce, zend_long long_key, zend_string *string_key)
```

Comment:
I wonder if it might be better to return `zend_result` here to indicate that this needs a `RETURN_THROWS();` if the constant update fails.

Reply:
Maybe this should also accept and handle `try`, so that this doesn't have to be handled manually.

Reply:
+1 for returning a `zend_result`. We could also comment that the caller must check `EG(exception)` in case of failure.

Reply:
@arnaud-lb Thanks for the review! I applied the suggestions, let me know if you're happy with it.